modules/Material/Toolbar.qml

```diff
@@ -153,7 +153,7 @@ View {
 
     Row {
         id: windowControls
-
+        visible: clientSideDecorations
```

Comment:
I wouldn't worry about it being unrelated to the rest of the pull request. Code reviews are supposed to improve code quality, not slow us down by forcing us to go through extra work and processes.